### PR TITLE
Add a Buster build.

### PR DESCRIPTION
**Pull request by dlorenc**
Switch the structure_test over to yaml.

### CODE REVIEW

**Comment by dlorenc** [comment]
Ref #79

File: reproducible/BUILD [hunk 1]
```diff
@@ -32,19 +32,39 @@ pkg_tar(
...
-    base = ":debootstrap_base",
+    base = ":jessie_base",
+    tars = [
+        ":overlay.tar",
+    ],
+)
+
+docker_build(
+    name = "debian9",
+    base = ":buster_base",
```
Comment (sharifelgamal) [inline reproducible/BUILD]: debian9 is stretch, debian10 is buster. we should have both, but stretch is GA and is obviously the higher priority.

Comment (dlorenc) [reply reproducible/BUILD]: D'oh. Added both.

File: reproducible/BUILD [hunk 2]
```diff
@@ -32,19 +32,58 @@ pkg_tar(
...
+structure_test(
+    name = "debian9_test",
+    config = "//tests:debian_test.yaml",
+    image = ":debian9",
+)
+
+structure_test(
+    name = "debian10_test",
+    config = "//tests:debian_test.yaml",
+    image = ":debian9",
```
Comment (sharifelgamal) [inline reproducible/BUILD]: :debian10

**Pull request merged**